.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,17 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
+contact_links:
+- name: 🙋  Usage Questions
+  url: https://github.com/osl-incubator/cookiecutter-python/discussions
+  about: |
+    Use Cookiecutter's GitHub Discussions to ask "How do I do X with Cookiecutter?".
```

@ayeankit I'd suggest explicitly using `cookiecutter-python` so that the users do not get confused between this library and the actual `cookiecutter`
```suggestion
    Use cookiecutter-python's GitHub Discussions to ask "How do I do X with cookiecutter-python?".
```